libdevcore/CommonData.h

```diff
@@ -172,9 +179,9 @@ inline std::string toHex(u256 val, HexPrefix prefix = HexPrefix::DontAdd)
 	return (prefix == HexPrefix::Add) ? "0x" + str : str;
 }
 
-inline std::string toCompactHex(u256 val, HexPrefix prefix = HexPrefix::DontAdd, unsigned _min = 0)
+inline std::string toCompactHex(u256 val, HexPrefix prefix = HexPrefix::DontAdd, PrintZero _min = PrintZero::DontPrint)
```

Comment:
there is a person who constantly asking me to change hex format in our tests. I've changed it here to PrintZero::DontPrint because that is what the meaning of that "1" was in the code. 
but the changes are not final.